07_deploy_masters.sh

```diff
@@ -79,5 +79,11 @@ while [ "$NODES_ACTIVE" -ne 3 ]; do
   sleep 10
   NODES_ACTIVE=$(oc --config ocp/auth/kubeconfig get nodes | grep "master-[0-2] *Ready" | wc -l)
 done
+
+# disable NoSchedule taints for masters until we have workers deployed
+for num in 0 1 2; do
+  oc --kubeconfig ocp/auth/kubeconfig adm taint nodes master-${num} node-role.kubernetes.io/master:NoSchedule-
+done
+
```

Comment:
Will this be persistent, or will the machine config mean the taint gets reapplied?

Reason for asking is the openshift/installer docs describe doing this by adding a manifest with a modified MachineConfig that removes the taint:

https://github.com/openshift/installer/blob/master/docs/user/customization.md#control-plane-with-no-taints

Perhaps we should do that instead, based on the manifest changes proposed in https://github.com/metalkube/dev-scripts/pull/127 ?

Reply:
While ideally we can follow docs, I got advice from @eparis that we should not use that approach.

I think the proposed approach is fine for now, until OpenShift defines another process for us that they recommend we adopt.

Reply:
I know little about the right approach, but being lazy I will assume for now that Russell knows better. ;) Let me know if we indeed should switch to modified manifests.

@russellb do you have details on why we should not use manifests yet?

Reply:
editting the systemd unit file which is used to launch the kubelet to prevent it from setting the taint will come back and bite us at some indeterminate time in the future when that systemd unit file needs to be changed by OpenShift, but your custom version will still be there overwriting what we need to change. I really need to get that out of the docs.

The taint will only re-appear if the node object is deleted from the API server (and re-created by restarting the kubelet on that machine).

You can save yourself a for loop by doing:
`oc adm taint node -l node-role.kubernetes.io/master node-role.kubernetes.io/master:NoSchedule-`

Reply:
ok, sound like we're good to go with this @booxter wanna remove the forloop as suggested or merge as it?

Reply:
I would remove but since it's merge already I don't think we should be bothered much.